src/components/profile/ProfileDetails.vue

```diff
@@ -50,8 +56,41 @@ export default defineComponent({
   setup() {
     const profile: Profile = reactive(formPersonalDetails as Profile);
 
+    const labelPaymentState = computed(() => {
+      switch (profile.paymentState) {
+        case PaymentState.paidByOrganization:
+          return i18n.global.t('profile.labelPaymentStatePaidByCompany');
+        case PaymentState.paid:
+          return i18n.global.t('profile.labelPaymentStatePaid');
+        default:
+          return i18n.global.t('profile.labelPaymentStateNotPaid');
+      }
+    });
+
+    const iconPaymentColor = computed(() => {
+      return profile.paymentState === PaymentState.paid ||
+        profile.paymentState === PaymentState.paidByOrganization
+        ? 'green'
+        : 'red';
+    });
+
+    const iconPaymentState = computed(() => {
+      return profile.paymentState === PaymentState.paid ||
```

Comment:
Alternatively you can use something like this:

 ```ts
`[PaymentState.paid, PaymentState.paidByOrganization].includes(profile.paymentState)
```

Reply:
Refactored using includes.